presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -704,4 +704,34 @@ public void testTableStatsSystemTable()
         // cleanup
         assertUpdate("DROP TABLE test_table_stats");
     }
+
+    @Test
+    public void testAlterTable()
+            throws InterruptedException
```

Comment:
If a unit test method needs checked exceptions, we normally just say `throws Exception`.